action.yml

```diff
@@ -36,7 +36,7 @@ runs:
         key: sonar-scanner-cli-${{ inputs.scannerVersion }}-${{ runner.os }}-${{ runner.arch }}
     - name: Install Sonar Scanner CLI
       if: ${{ env.NO_CACHE == 'true' || steps.sonar-scanner-cli.outputs.cache-hit != 'true' }}
-      run: ${GITHUB_ACTION_PATH}/install-sonar-scanner-cli.sh
+      run: ${GITHUB_ACTION_PATH}/scripts/install-sonar-scanner-cli.sh
```

Comment:
Similary I would consider reusing `download.sh` content when installing-sonar-scanner-cli in later commits. And merging the two approaches.

Reply:
Idea captured in this JIRA ticket: https://sonarsource.atlassian.net/browse/SQSCANGHA-69